complex/cargo/list/src/lib.rs

```diff
@@ -374,6 +411,76 @@ pub unsafe extern "C" fn list_manager_create_item(manager: *mut ListManager, ite
     manager.create_item(&item);
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn list_manager_create_item_direct(manager: *mut ListManager, name: *const c_char, due_date: *const time_t) {
+    let name = c_char_to_string(name);
+    log::d(&format!("Creating item: {:?}, {:?}, {:?}", name, due_date, manager)[..]);
+
+    let manager = &mut*manager;
+    let mut item = Item::new();
+
+    item.name = name;
+    item.due_date = Some(Timespec::new(due_date as i64, 0));
+
+    manager.create_item(&item);
```

Comment:
It might make more sense to update `create_item` to take the item properties as arguments and not have to have this intermediary step of creating an `Item` so we can create an `Item`